frontends/web/src/routes/account/account.tsx

```diff
@@ -222,6 +239,13 @@ export function Account({
         <Status hidden={!hasCard} type="warning">
           {t('warning.sdcard')}
         </Status>
+        <Dialog open={insured && uncoveredFunds.length !== 0} medium title={t('account.warning')} onClose={() => setUncoveredFunds([])}>
+          <MultilineMarkup tagName="p" markup={t('account.uncoveredFunds', {
+            name: account?.name,
+            uncovered: uncoveredFunds,
+          })}/>
+          <A href="#">{t('account.uncoveredFundsLink')}</A>
```

Comment:
small nit: using anchor with `href="#"` is usually indication that we should use a button element.

in BBApp iirc we use `<A>` component to open links in default browser.

Reply:
yeah, this should be a link, we just don't have the guide page yet! Maybe I can add a //TODO comment to fix before merging into master